AirportDictionary/AirportDictionary/Pages/Index.cshtml.cs

```diff
@@ -20,47 +20,42 @@ public IndexModel(ILogger<IndexModel> logger)
             _logger = logger;
         }
 
-        public void OnGet()
+        public JsonResult OnGet()
         {
 
-            using (var webClient = new System.Net.WebClient())
-            {
-
-                //IDictionary<long, QuickTypeCity.Welcome> allCities = new Dictionary<long, QuickTypeCity.Welcome>();
-
-                string cityJson = webClient.DownloadString("https://pkgstore.datahub.io/core/world-cities/world-cities_json/data/5b3dd46ad10990bca47b04b4739a02ba/world-cities_json.json");
+                string cityJson = getJsonData("https://pkgstore.datahub.io/core/world-cities/world-cities_json/data/5b3dd46ad10990bca47b04b4739a02ba/world-cities_json.json");
                 QuickTypeCity.Welcome[] allCity = QuickTypeCity.Welcome.FromJson(cityJson);
 
-                ViewData["Welcome"] = allCity;
-
-                //foreach (QuickTypeCity.Welcome welcome in allCity)
-                //{
-                //    allCities.Add(welcome.Geonameid, welcome);
-
-                //}
-
-                // QuickTypeAirport.Airport[] allAirport = QuickTypeAirport.Airport.FromJson(airportJson);
+                
 
+               
 
-                string airportJson = webClient.DownloadString("https://pkgstore.datahub.io/core/airport-codes/airport-codes_json/data/552f9a534537e5a2f7a8b0584dad7e77/airport-codes_json.json");
+                string airportJson = getJsonData("https://pkgstore.datahub.io/core/airport-codes/airport-codes_json/data/552f9a534537e5a2f7a8b0584dad7e77/airport-codes_json.json");
```

Comment:
While method names are camelCase in Java, they are generally PascalCase in C#.  Change this to GetJsonData.